Tests/MintTests/MintTests.swift

```diff
@@ -222,4 +226,64 @@ class MintTests: XCTestCase {
             try mint.install(package: PackageReference(repo: "yonaskolb/simplepackage", version: "compile_error"))
         }
     }
+
+    func testListSimpleMintFile() throws {
+        mint.mintFilePath = simpleMintFileFixture.absolute()
+
+        try mint.bootstrap(link: true)
+        let cache = try Cache(
+            path: mint.packagesPath,
+            metadata: mint.readMetadata(),
+            linkedExecutables: mint.getLinkedExecutables()
+        )
+
+        XCTAssertEqual(cache.list, """
+          SimplePackage
+            - 4.0.0 *
+        """
+        )
+    }
+
+    func testListComplexMintFile() throws {
+        mint.mintFilePath = complexMintFileFixture.absolute()
+
+        try mint.bootstrap(link: true)
+        let cache = try Cache(
+            path: mint.packagesPath,
+            metadata: mint.readMetadata(),
+            linkedExecutables: mint.getLinkedExecutables()
+        )
+
+        XCTAssertEqual(cache.list, """
+          SimplePackage (https://github.com/yonaskolb/SimplePackage.git)
+            - 4.0.0
+          SimplePackage (https://github.com/acecilia/SimplePackage.git)
```

Comment:
If you have other account you prefer to rely on, instead of mine, I can change this

Reply:
Totally fine, if it sticks around 👍